content/docs/for-buildpack-authors/how-to/write-buildpacks/clear-env.md

```diff
@@ -3,8 +3,23 @@ title="Clear the buildpack environment"
 weight=99
 +++
 
+During the `detect` and `build` phases, the lifecycle MUST provide as environment variables any user-provided files in `<platform>/env/` with environment variable names and contents matching the file names and contents.
+
 <!--more-->
 
-This page is a stub! The CNB project is applying to [Google Season of Docs](https://developers.google.com/season-of-docs/docs/timeline) to receive support for improving our documentation. Please check back soon.
+The `lifecycle` MUST also provide as environment variables any operator-provided files in `<build-config>/env` with environment variable names and contents matching the file names and contents. This applies for all values of `clear-env` or if `clear-env` is undefined in `buildpack.toml`.
+
+* When `clear-env` in `buildpack.toml` is set to `true` for a given buildpack, the lifecycle MUST NOT set user-provided environment variables in the environment of `/bin/detect` or `/bin/build`.
+* When `clear-env` in `buildpack.toml` is not set to `true` for a given buildpack, the lifecycle MUST set user-provided environment variables in the environment of `/bin/detect` or `/bin/build` such that:
+
+  * For layer path environment variables, user-provided values are prepended before any existing values and are delimited by the OS path list separator.
```

Comment:
Let's move this stuff to the "specify env" section. It's not super relevant for clear env.

Reply:
done